rust/datafusion/src/physical_plan/parquet.rs

```diff
@@ -99,18 +154,38 @@ impl ParquetExec {
                 .collect(),
         );
 
+        // sum the statistics
+        let mut num_rows = 0;
+        let mut total_byte_size = 0;
+        for part in &partitions {
+            num_rows += part.statistics.num_rows.unwrap_or(0);
+            total_byte_size += part.statistics.total_byte_size.unwrap_or(0);
+        }
+        let statistics = Statistics {
+            num_rows: if num_rows == 0 {
+                None
```

Comment:
Why map rows and byte size to None here and not `Some(0)`?

Reply:
I think if the result is 0, it is best to know that there are 0 records/bytes.

Reply:
Yes, that was a bit sloppy. This is now fixed.